### PR TITLE
models generated

### PR DESCRIPTION
**Pull request by julescarriere**

### CODE REVIEW

File: app/models/toilet.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Toilet < ApplicationRecord
+  belongs_to :user
```
Comment (lorrainefav) [inline app/models/toilet.rb]: has_many :bookings, dependent: :destroy

**Comment by ya33** [comment]
on va annuler cette branch! sorry!

**Comment by GauthierValin** [comment]
Ok

**Comment by julescarriere** [comment]
dacc

**Comment by GauthierValin** [comment]
shut up Jules

**Pull request closed without merging**